### DISCUSSION TITLE
Feature Request: Use of Notes in extension

### DISCUSSION THREAD

**Discussion by Damieee** [Dev-Motion/paige#76] [Feature Request]
Thank you, Team Devio, for this fantastic extension. It has proven incredibly useful. I've been contemplating the idea of incorporating a note feature into the extension to enhance productivity and organization. This way, I can have all my essential tools in one comprehensive extension.

**Comment by Code-Victor** [comment]
@Damieee, Thanks for Suggesting this, we will add it to our list of features to release in the future👍🏾

Btw, what features would you love the notes to have?
